modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, the eBPF agent processes flows at a degraded performance. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+[NOTE]
+====
+Increasing the `cacheMaxFlows` may increase the memory usage of the eBPF agent. 
+====
+.Prerequisite
+* You must have eBPG metrics enabled by setting the `spec.agent.type.ebpf.metrics.enable` value to `true`. For more information, see "Enabling eBPF metrics".
+
+
+.Procedure
+
+. In the web console, navigate to *Operators* → *Installed Operators*.
```

Comment:
I've been told not to use the → symbol, and instead use `->` because it can cause issues. I know `->` does get translated into the symbol, but...this is just what I was told.

Reply:
Hmm very interesting this was here actually because I usually use ->. I may have copied this step from HTML and forgot to change the icon. good catch!